x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/crawl_custom_settings_flyout/url_combo_box.scss

```diff
@@ -0,0 +1,5 @@
+.urlComboBox {
+  .euiComboBox__inputWrap {
+    min-height: 60px;
```

Comment:
Only suggestion here would be to use a `rem` value instead of the `px`... Should work out to `3.75rem`

Reply:
So I got `60px` because the default row height of a row in an `EuiComboBox` is [29px](https://github.com/elastic/eui/blob/main/src/components/combo_box/combo_box_options_list/combo_box_options_list.tsx#L114) + [1px](https://github.com/elastic/eui/blob/main/src/components/combo_box/combo_box_options_list/combo_box_options_list.tsx#L441) = 30px, which I doubled so it defaults to two rows.  There's no way to specifically set the default number of rows for `EuiComboBox` unfortunately.  With that in mind should I still convert to `rem`? Should I leave as-is but write a comment? I'll follow your call